assets/javascripts/kitten/components/graphics/logos/kisskissbankbanklogo/index.js

```diff
@@ -20,7 +22,7 @@ export const KissKissBankBankLogo = ({
   })
 
   const viewBox = {
-    x: tiny ? TINY_WIDTH : DEFAULT_WIDTH,
+    x: small || tiny ? SMALL_WIDTH : DEFAULT_WIDTH,
```

Comment:
J'ajouterais ça pour plus de lisibilité

```suggestion
    x: (small || tiny) ? SMALL_WIDTH : DEFAULT_WIDTH,
```